packages/homepage/src/components/TeamMember.js

```diff
@@ -142,6 +143,12 @@ const people = [
     pic: james,
     color: '#6200ff',
     team: 'Support',
+  },
+  {
+    name: 'Maria Clara',
+    pic: maria,
+    color: 'F24E62',
```

Comment:
I think you need the # here or the highlight color won't work

Reply:
@gsimone thanks for the catch! fixed it here: 48061a3